section2/reflection.md

```diff
@@ -2,14 +2,35 @@
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+   At no point did I feel _uncomfortable_, per se. I felt confident in running the node command even though it took me a couple of tries from memory. The vague instructions helped me with memory recall and understanding how each section/skill builds on itself. I feel like I've gained insight into my own learning style, my strengths and weaknesses, and how I can leverage all of them to succeed. It's empowering, actually.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+   * A conditional statement evaluates `true` or `false` and commonly uses comparison operators such as `===`, `!==`, `<`, `<=`, `>`, and `>=`.
+   * A daily life example would be to ask myself, "Are the dishes clean?" If yes (or `true`), then great! There's no work to be done. If no (or `false`), then it looks like I'll be doing some dishes.
+   * A web application that I use often is Spotify. An example of a conditional statement there would be the now playing function. If the play button is pressed, play music. If the play button is not pressed, do not play music.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+   You can add multiple conditions to an `if...else` statement by using `else if` statements where there are multiple conditions and code blocks that will run depending on if the previous statement is evaluated as `true` or `false`.
+
+   The program reads them line by line. If the initial `if` conditional statement is true, we stop there. If it's `false` then we move to the next code block to evaluate the second condition. That code block will run if evaluated as `true`. If it results in `false` we move again to the next code block to evaluate a third condition, and so on and so forth.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+   * If I wanted to check if two conditions are met, I would use the AND logical operator `&&`
+   * If I wanted to check that 1 of 2 conditions are met, I would use the OR logical operator `||`
+
+7. What questions do you still have about `if` statements and/or functions?
+
+   Why do we use a space after the `if` statement but not after `console.log`?
```

Comment:
What a great question! I have never thought about it or been asked that one before.

I would say that goes to the conventions of JavaScript - both would work with or without the space. `console.log` is a function (you'll get into those in Mod 1) and the convention for calling a function includes those parenthesis immediately after the function name.

And for whatever reason, the convention for the if statements calls for a space - I would assume readability.

This feels like a "because it's the way it is" answer!